### DISCUSSION TITLE
What about layouts?

### DISCUSSION THREAD

**Discussion by hexwit** [wimdeblauwe/wimdeblauwe.com#3] [Q&A] [answered]
Hi, 

I have read https://www.wimdeblauwe.com/blog/2022/07/03/thymeleaf-live-reload-with-npm-scripts/ and 
seems to be that solution will not work with layouts. I mean, I see there only live reload, but who is building the 
template itself? What If i need to add some variables to template, layouts, localisation. 

May be I miss something, but as for now, it is like I've got live reload just for a plain html file. 
Could you please clarify what I miss?

**Comment by wimdeblauwe** [comment, marked as answer]
You need to run the Spring Boot application using the `local` profile and run `npm run build && npm run watch` to have the full live reload experience. There is now also the ttcli command line tool to quickly setup a project, see https://github.com/wimdeblauwe/ttcli
